build

```diff
@@ -1,8 +1,16 @@
 #!/bin/sh -e
 
+CDIR=$(cd `dirname "$0"` && pwd)
+cd "$CDIR"
+
 # set some environment variables
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
+if [ ! -h gopath/src/${REPO_PATH} ]; then
```

Comment:
probably move https://github.com/coreos/etcd/blob/master/test#L224-L227 into `function etcd_setup_gopath { ... }` and do `(echo "$0" | grep "build$" > /dev/null) && etcd_setup_gopath && etcd_build || true` at the end of this file?